src/component/Banner.tsx

```diff
@@ -332,24 +332,22 @@ class Banner extends Component<Props, State> {
                                     'commercialCmpCopy-variant' ? (
                                         <>
                                             <p>
-                                                We use your information –
-                                                collected through cookies and
-                                                similar technologies – to
-                                                improve your experience on our
-                                                site, analyse how you use it,
-                                                show you personalised
-                                                advertising and share that
-                                                information with{' '}
+                                                We and our{' '}
                                                 <button
                                                     css={buttonAsLinkStyles}
                                                     onClick={() => {
-                                                        onOptionsClick();
+                                                        onOptionsClick(true);
```

Comment:
Maybe a comment to tell what true means?

Reply:
The function signature is declared in line 290.